pkg/config/constants.go

```diff
@@ -12,6 +12,14 @@
 
 package config
 
+// Labels which should be used for controller related objects
+var ControllerAppLabels = func() map[string]string {
```

Comment:
To avoid occasional change by clients since we can't put map as const. So I came with function that will return always new instance which clients will be able to fill in with extra items if needed.
@amisevsk WDYT? Can you suggest another way to archive it?

Reply:
That makes sense, it just looked like a strange construct at first glance.